en/datatypes/email.adoc

```diff
@@ -0,0 +1,62 @@
+= Email! datatype
+:toc:
+:numbered:
+
+== Abstract
+
+`Email!` values allow for directly expressing email addresses. `foo@bar.net`
+
+`Email!` is a member of the following typsets: `any-string!`, `series!`
+
+== Creation
+
+`Email!` values can be created using literal syntax, or at runtime using a `make` constructor or a `to` conversion.
+
+```red
+>> f: make email! "foo"
+== foo
+
+>> type? f
+== email!
+```
+
+```red
+>> b: to email! 'bar
+== bar
+
+>> type? b
+== email!
+```
+
+== Literal syntax
+
+The `@` symbol, and zero or more additional characters.
```

Comment:
Let's say there have to be chars before the `@`, even though the lexer allows them not being there today. That syntax may become a `ref!` type.